src/Symfony/Component/Yaml/Yaml.php

```diff
@@ -89,7 +90,7 @@ public static function parse($input, $flags = 0)
      *
      * @return string A YAML string representing the original PHP array
      */
-    public static function dump($array, $inline = 2, $indent = 4, $flags = 0)
+    public static function dump($array, $inline = 2, $indent = 4, $absIndent = 0,$flags = 0)
```

Comment:
Hi,

I don't know php.... How do I carry that value over without passing it as
argument in recursion?

An hint will suffice!

Will write tests once you approve at least what I am doing...

Merc.
On 24 Feb 2016 6:20 pm, "Christophe Coevoet" <notifications@github.com>
wrote:

> In src/Symfony/Component/Yaml/Yaml.php
> <https://github.com/symfony/symfony/pull/17912#discussion_r53918375>:
>
> > @@ -89,7 +90,7 @@ public static function parse($input, $flags = 0)
> >       *
> >       * @return string A YAML string representing the original PHP array
> >       */
> > -    public static function dump($array, $inline = 2, $indent = 4, $flags = 0)
> > +    public static function dump($array, $inline = 2, $indent = 4, $absIndent = 0,$flags = 0)
>
> Do we actually need to expose it to the outside here ? This makes the
> signature harder to use and understand (and a pain for backward
> compatibility).
>
> Btw, the fact that you are adding the argument before an existing one
> breaks all usages of the flags.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/symfony/symfony/pull/17912/files#r53918375>.
>